src/OrchardCore.Modules/OrchardCore.Lists/RemotePublishing/MetaWeblogHandler.cs

```diff
@@ -153,15 +155,31 @@ private async Task<XRpcStruct> MetaWeblogNewMediaObjectAsync(string userName, st
 
         var directoryName = Path.GetDirectoryName(name);
         var filePath = _mediaFileStore.Combine(directoryName, Path.GetFileName(name));
-        Stream stream = null;
+        MemoryStream stream = null;
+        (string outputPath, Stream outputStream) output = (null, null); // Initialize the tuple to avoid CS0170
```

Comment:
I would prefer to use `default` instead:

```suggestion
        (string outputPath, Stream outputStream) output = default;
```